views/style.css

```diff
@@ -76,18 +143,63 @@ a.navtab {
   flex-direction: column;
   text-align: center;
   color: white;
-  background: linear-gradient(#430577, #a3ffa0)
+  background: linear-gradient(to right, #8360c3, #2ebf91);
 }
 
 #kubercade_nexus h1 {
-  font-size: 32pt;
+  text-shadow: 4px 4px 2px rgb(22, 22, 22);
 }
 
 #kubercade_nexus h4 {
-  font-size: 24pt;
+  text-shadow: 2px 2px 2px rgb(22, 22, 22);
 }
 
 #kubercade_nexus a {
   font-size: 18pt;
   color: white;
+}
+
+/*** chat ***/
+
+/* yt colors for when chat has a UI
+  background dark: #191919
+  background mid: #282828
+  ui dark: #333333
+  ui buttons/hints: #686868 */
+
+#chat_window {
+  background-color: #191919;
+  color: white;
+}
+
+/*** scores ***/
+#high_scores_container {
+  background-color: #282828;
+  height: 100%;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+}
+
+#high_scores_container h1 {
+  text-shadow: 2px 2px 1px black;
+}
+
+#high_scores_container .scores_list {
+  width: 100%;
+  max-width: 1200px;
+  overflow-y: auto;
+}
+
+#high_scores_container .individual_score {
+  display: grid;
+  grid-template-columns: 2fr 4fr 2fr 5fr;
+  margin-bottom: 6px;
+  text-shadow: 0px 1px 1px white;
```

Comment:
I think the shadow color looks slightly strange with some of the text colors (like scores 2 and 5 in the [screenshot](https://screenshot.googleplex.com/r3kWwkaDSeb) - it might also just be the spacing though.

Reply:
Removed https://github.com/knative-portability/Kubercade/pull/17/commits/56fee0f63c13049a243eaaafdde8d2fd34ddfa47